src/component/axis/CartesianAxisView.ts

```diff
@@ -135,6 +139,37 @@ const axisElementBuilders: Record<typeof selfBuilderAttrs[number], AxisElementBu
             tickModel: splitLineModel
         });
 
+        if (ticksCoords.length > 2 && (axis.scale as any).getInterval instanceof Function) {
+            const interval = (axis.scale as IntervalScale).getInterval() || null;
+            const precision = (axis.scale as IntervalScale).getIntervalPrecision instanceof Function
+                ? (axis.scale as IntervalScale).getIntervalPrecision()
+                : null;
+
+            const showMinLine = splitLineModel.get('showMinLine');
+            if (showMinLine === false
+                || (showMinLine === 'auto'
+                    && roundNumber(
```

Comment:
Using `roundNumber` to check if it is added as the *nice* extent is not accurate.